src/call.cpp

```diff
@@ -941,6 +941,38 @@ bool call::connect_socket_if_needed()
 
         sipp_customize_socket(call_socket);
 
+        if (peripsocket) {
+            struct sockaddr_storage saddr;
+            char peripaddr[256];
+            char *tmp = peripaddr;
+            getFieldFromInputFile(ip_file, peripfield, NULL, tmp);
```

Comment:
What good does the addition of `char *tmp = ...` do? Looks to me like `peripaddr` could simply be passed twice as args 2 and 4..

(And I believe we settled on `char*`.)